weasyl/search.py

```diff
@@ -99,6 +90,15 @@ def add_nonempty(s, item):
             tag = d.get_search_tag(criterion)
             add_nonempty(self.required_includes, tag)
 
+    def __bool__(self):
```

Comment:
Sadly, python 2 and 3 differ here. This needs to be available as both `__bool__` and `__nonzero__`.